src/components/Person/Person.jsx

```diff
@@ -1 +1,53 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {`${partnerName} is my ${partner}`}
+          </p>
+        )
+        : (
+          <p className="Person__partner">I am not married</p>
+        )}
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  age: null,
```

Comment:
Useless as for me. If `age` is not passed, then it will be in the conditions give `false`. Rendering `Person__age` won't happen in that case.